modules/035-cni-flannel/images/flanneld/werf.inc.yaml

```diff
@@ -21,38 +21,67 @@ shell:
         done
       done
 ---
-artifact: {{ .ModuleName }}/entrypoint-artifact
-from: {{ $.Images.BASE_GOLANG_21_ALPINE_DEV }}
-git:
-- add: /{{ .ModulePath }}modules/035-cni-flannel/images/flanneld/entrypoint
+image: {{ .ModuleName }}/{{ .ImageName }}-entrypoint-artifact
+from: {{ .Images.BASE_GOLANG_23_ALPINE_DEV }}
```

Comment:
Please use ASE_GOLANG_23_ALPINE and install packages thru proxy

Reply:
Fixed